pinot-tools/src/main/resources/examples/minions/batch/airlineStats/airlineStats_offline_table_config.json

```diff
@@ -0,0 +1,45 @@
+{
+  "tableName": "airlineStats",
+  "tableType": "OFFLINE",
+  "segmentsConfig": {
+    "timeColumnName": "DaysSinceEpoch",
+    "timeType": "DAYS",
+    "segmentPushType": "APPEND",
+    "segmentAssignmentStrategy": "BalanceNumSegmentAssignmentStrategy",
+    "replication": "1"
+  },
+  "tenants": {},
+  "tableIndexConfig": {
+    "loadMode": "MMAP"
+  },
+  "metadata": {
+    "customConfigs": {}
+  },
+  "ingestionConfig": {
+    "batchIngestionConfig": {
+      "segmentIngestionType": "APPEND",
+      "segmentIngestionFrequency": "DAILY",
+      "batchConfigMaps": [
+        {
+          "inputDirURI": "s3://my.s3.bucket/batch/airlineStats/rawdata/",
+          "fs.className": "org.apache.pinot.plugin.filesystem.S3PinotFS",
```

Comment:
aren't these expected to be "input.fs.className" and "input.fs.prop.region" ? Will this work without input/output prefix?

Reply:
it won't, updated the config.